compiler/rustc_monomorphize/src/partitioning/mod.rs

```diff
@@ -149,7 +150,10 @@ fn get_partitioner<'tcx>(tcx: TyCtxt<'tcx>) -> Box<dyn Partitioner<'tcx>> {
 
     match strategy {
         "default" => Box::new(default::DefaultPartitioning),
-        _ => tcx.sess.fatal("unknown partitioning strategy"),
+        _ => {
+            let error_message = "unknown partitioning strategy".to_string();
+            tcx.sess.emit_fatal(FatalError { error_message: error_message.clone() });
```

Comment:
This should really be a `UnknownPartitioningStrategy` error that has this message in the Fluent, rather than a `FatalError` type that just has the English string. Likewise with other uses of `{Span,}FatalError`.

Reply:
Will do 👍🏻

Reply:
Done with `UnknownPartitionStrategy` and `SymbolAlreadyDefined`.